exercises/J-functions/exercise.js

```diff
@@ -1,7 +1,7 @@
 function halve(number) {
-  // complete the function here
+  return Number*.5
```

Comment:
variables are case-sensitive; in this case you want to refer to the `number` variable; but instead you're referring to something called `Number` (which, incidentally, is a thing that exists in JS)